braindecode/datautil/preprocess.py

```diff
@@ -313,10 +318,10 @@ def filterbank(raw, frequency_bands, drop_original_signals=True,
             for old_name in filtered.ch_names})
         all_filtered.append(filtered)
     raw.add_channels(all_filtered)
-    # reorder channels by frequency band
-    chs_by_freq_band = [
-        ch for i in range(len(original_ch_names))
-        for ch in raw.ch_names[i::len(original_ch_names)]]
-    raw.reorder_channels(chs_by_freq_band)
+    if not order_by_frequency_band:
+        chs_by_freq_band = [
+            ch for i in range(len(original_ch_names))
+            for ch in raw.ch_names[i::len(original_ch_names)]]
+        raw.reorder_channels(chs_by_freq_band)
```

Comment:
Can you add a comment directly above what this code does? I don't get it from just having a short look.